simple_https_server/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(non_upper_case_globals, dead_code)]
+#![allow(non_upper_case_globals, clippy::zero_prefixed_literal)]
```

Comment:
Have you seen this https://github.com/cryspen/bertie/issues/53? I will accept the PR as is but strongly suggest to not do this. There is a reason clippy lints against that.

Reply:
I don't consider this a useful lint and it's just noise.